packages/TorneloScoresheet/src/util/env.ts

```diff
@@ -1,4 +1,8 @@
-import { TORNELO_URL, MAX_PAST_GAME_STORED } from '@env';
+import { TORNELO_URL, MAX_PAST_GAME_STORED, EMAIL_API_ENDPOINT } from '@env';
 
 export const torneloUrl = TORNELO_URL;
 export const maxPastGameStored = MAX_PAST_GAME_STORED;
+export const emailEndpoint = (arbiterId: string): string => {
+  console.log(EMAIL_API_ENDPOINT);
```

Comment:
there are a few console.log statements here and in emailUtils. Are they meant to be there?